cmd/gen-go-sample/inittree.go

```diff
@@ -511,3 +515,21 @@ func snakeToCamel(s string) string {
 	}
 	return sb.String()
 }
+
+func printCommentLines(w *bufio.Writer, comment string, ind int) {
+	if comment == "" {
+		return
+	}
+	for _, c := range strings.Split(strings.TrimRight(comment, "\n"), "\n") {
+		indent(w, ind)
+		w.WriteString("// ")
+		w.WriteString(c)
+		w.WriteByte('\n')
+	}
+}
+
+func indent(w *bufio.Writer, ind int) {
+	for i := 0; i < ind; i++ {
+		w.WriteByte('\t')
```

Comment:
+1 for spaces instead of tabs for consistency.

Minor aside: how important is it to be diligent about formatting? Is it better to just do best-effort and let gofmt do the heavy lifting?

Reply:
> how important is it to be diligent about formatting?

While most of the generation was written by Mike D., in gapic-generator-go we try to stay roughly aligned with proper formatting, including indentation of code blocks. It makes the generation code much easier to read/mentall parse. Something like an empty curly-brace pair being on two lines isn't a big deal, it gets fixed-up by gofmt.

Reply:
The entire sample gen project uses tabs to indent.

And I found this: 

> Indentation
> We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.

source: https://stackoverflow.com/questions/19094704/indentation-in-go-tabs-or-spaces